src/main/java/pl/coderstrust/accounting/controller/InvoiceController.java

```diff
@@ -153,4 +161,21 @@ public void removeInvoice(@RequestBody int id) {
     invoiceService.updateInvoice(invoice);
     return ResponseEntity.ok().build();
   }
-}
\ No newline at end of file
+
+  @RequestMapping(value = "pdf/{id}", method = RequestMethod.GET, produces = MediaType.APPLICATION_PDF_VALUE)
+  @ApiOperation(value = "Returns invoice in pdf format")
+  public ResponseEntity<byte[]> invoicePdf(@PathVariable("id") int id) {
```

Comment:
I would rename it to getInvoiceAsPdf - again cosmetic change but making code easier to understand

Reply:
done